package.json

```diff
@@ -24,15 +24,19 @@
         "build": "npm run clean && npm run generate && cross-env NODE_ENV=production npm run build:library",
         "build:library": "rollup -c",
         "build:storybook": "build-storybook -s .storybook/assets",
+        "build:styles": "tailwindcss --jit --output ./dist/styles.css",
         "clean": "rimraf dist",
         "component:create": "ts-node scripts/createNewComponent.ts",
         "generate": "ts-node scripts/generateExports.ts",
         "lint": "npm run lint:ts",
         "lint:fix": "npm run lint:ts:fix",
         "lint:ts": "eslint 'src/**/*.{ts,tsx}'",
         "lint:ts:fix": "eslint --fix 'src/**/*.{ts,tsx}'",
+        "pretest": "npm run build:styles",
         "test": "cypress run-ct",
+        "pretest:browser": "npm run build:styles",
         "test:browser": "cypress open-ct",
+        "pretest:file": "npm run build:styles",
```

Comment:
Do you actually need it? As `rollup` and `vite` take care of that

Reply:
the problem is with this test https://github.com/Frontify/arcade/pull/42/commits/af3adae6a9202b4b8e21d6ed261d81245ad7e397#diff-c89024bae78b39f1cffe9f214ed808043aae2dcab2e3d0e7307b2aa545b29c04R69 atm if i do not build and add the css via `{ cssFile: "./dist/styles.css" },` while `mount` the component, all the tested components are unstyled and therefore my `tab` test fails because the input field is not hidden... i already invested some time googling for a "global" solution so that cypress would do the magic but didn't find any, so if you know of one i'm open for inputs 🙂  but for now i would go with this...

Reply:
tweaked it a little b482e65 and git rid off the `build:styles` script